docs/apache-airflow/concepts/cluster-policies.rst

```diff
@@ -26,9 +26,9 @@ If you want to check or mutate DAGs or Tasks on a cluster-wide level, then a Clu
 
 There are three types of cluster policy:
 
-* ``dag_policy``: Takes a :class:`~airflow.models.dag.DAG` parameter called ``dag``. Runs at load time.
-* ``task_policy``: Takes a :class:`~airflow.models.baseoperator.BaseOperator` parameter called ``task``. Runs at load time.
-* ``task_instance_mutation_hook``: Takes a :class:`~airflow.models.taskinstance.TaskInstance` parameter called ``task_instance``. Called right before task execution.
+* ``dag_policy``: Takes a :class:`~airflow.models.dag.DAG` parameter called ``dag``. Runs at load time of the dag from DagBag.
```

Comment:
The rest of this doc is using the all-caps DAG nomenclature, I'd stick with that. Also, a link to the DagBag class might be helpful.
```suggestion
* ``dag_policy``: Takes a :class:`~airflow.models.dag.DAG` parameter called ``dag``. Runs at load time of the DAG from DagBag.
```